### DISCUSSION TITLE
Confused!!!

### DISCUSSION THREAD

**Discussion by OtusiChikwadoMark** [community/community#39736] [New to GitHub] [answered]
Please I'm new, I need guard on how to perform here...

**Comment by Akash1134** [comment]
👋 Welcome to the GitHub Community on Discussions, @OtusiChikwadoMark  Happy to have you here 🙂

To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776) and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912).

If you need any specific information on our products, let us know. Thanks!

**Comment by DSDmark** [comment, marked as answer]
Hey @OtusiChikwadoMark,

Change your category into **new to GitHub**.   

> Nice to meet you and welcome to this great community, we are here to help each other and try to make open source better. This is the discussions section if you want to discuss any. you can ask here, we will try to help.

Or why any chance if you are curious about, 

> what is GitHub, who is GitHub then I highly suggest you check this out [**What is GitHub?**](https://youtu.be/pBy1zgt0XPc).  its a video. so, just ***sit back and relax***

**Reply by OtusiChikwadoMark** [reply]
Oh thank you so very much for your response, i appreciate.
